### ISSUE TITLE
Annotations and download image

### ISSUE THREAD

**Issue by bhavesh-pandey** [OHIF/Viewers#700]
Hi Team,

Firstly great work with the new OHIF react viewer.

I was wondering when the annotate feature and download image feature will come as it's very important for sharing findings in a dicom image. When can we expect to see that?

Thanks

**Comment by dannyrb** [comment]
@bhavesh-pandey, a large portion of the development of the OHIF viewer comes from volunteers like you. While these features will likely eventually be implemented, I can't make any promises around when someone will decide to champion them. If you or your team would be interested in contributing, I can provide some guidance on where to get started and what we would be looking for in a PR? Alternatively, I can direct you toward individuals/teams that have historically done OHIF related contract work.

**Comment by bhavesh-pandey** [comment]
Yeah we would love to do that. If you can help us we can develop those features.

**Comment by swederik** [comment]
The Image Download UI from the previous version was here:

https://github.com/OHIF/Viewers/tree/v1.x/Packages/ohif-viewerbase/client/components/viewer/downloadDialog

The first step would be to port this UI to React and make it work with the Cornerstone extension. Later we would need to decide how this should be handled for e.g. the VTK plugin.

As for the Annotation tool, it should be easy enough to add it to the list of available tools, but we need to add some UI to enter the actual annotation text. This stuff was previously here: https://github.com/OHIF/Viewers/tree/v1.x/Packages/ohif-viewerbase/client/components/viewer/annotationDialogs

**Comment by swederik** [comment]
BTW, this is partly a duplicate of https://github.com/OHIF/Viewers/issues/674 so please search before creating new issues.
